operator/v1/types_network.go

```diff
@@ -80,6 +80,11 @@ type NetworkSpec struct {
 	// If not specified, sensible defaults will be chosen by OpenShift directly.
 	// Not consumed by all network providers - currently only openshift-sdn.
 	KubeProxyConfig *ProxyConfig `json:"kubeProxyConfig,omitempty"`
+
+	// exportNetworkFlows enables and configures the export of network flows from OVS
+	// by using protocols netFlow, sFlow or ipfix.
+	// +optional
```

Comment:
if not set, tell me what the default does.

Reply:
I can comment, but nothing. I simply won't export flows.

Reply:
done